entrypoint.sh

```diff
@@ -1,5 +1,33 @@
 #!/bin/bash
 
+# usage: file_env VAR [DEFAULT]
+#    ie: file_env 'XYZ_DB_PASSWORD' 'example'
+# (will allow for "$XYZ_DB_PASSWORD_FILE" to fill in the value of
+#  "$XYZ_DB_PASSWORD" from a file, especially for Docker's secrets feature)
+file_env() {
+	local var="$1"
+	local fileVar="${var}_FILE"
+	local def="${2:-}"
+	if [ "${!var:-}" ] && [ "${!fileVar:-}" ]; then
+		printf >&2 'error: both %s and %s are set (but are exclusive)\n' "$var" "$fileVar"
+		exit 1
+	fi
+	local val="$def"
+	if [ "${!var:-}" ]; then
+		val="${!var}"
+	elif [ "${!fileVar:-}" ]; then
+		val="$(< "${!fileVar}")"
+	fi
+	export "$var"="$val"
+	unset "$fileVar"
+}
+
+file_env "WAKAPI_PASSWORD_SALT" ""
+file_env "WAKAPI_DB_PASSWORD" ""
+file_env "WAKAPI_MAIL_SMTP_PASS"
+file_env "WAKAPI_SUBSCRIPTIONS_STRIPE_SECRET_KEY"
+file_env "WAKAPI_SUBSCRIPTIONS_STRIPE_ENDPOINT_SECRET"
```

Comment:
What do we need these for?

Reply:
To check if any of those env variables are present with the "_FILE"-suffix and load them from the given file if they are